src/library_webgl.js

```diff
@@ -3536,8 +3536,10 @@ var LibraryGL = {
   glBindVertexArrayOES: 'glBindVertexArray',
   glIsVertexArrayOES: 'glIsVertexArray',
 
+#if LEGACY_GL_EMULATION
```

Comment:
these functions can be moved to the separate `library_glemu.js`

Reply:
The could.. but I wasn't sure that the rule was here.. there seem to be other usages of LEGACY_GL_EMULATION in this file.   I will propose that as followup.

Reply:
Actually you are right.  I moved them as part of this PR.